dependencies.properties

```diff
@@ -0,0 +1,54 @@
+# This file is shared between gradle and bazel build scripts and exists to keep the dependencies
+# configuration between the two build systems in sync.
+
+# The properties format is the following:
+#   <dependency_type>.<artifact_name>=<value>
+
+# Target workspace name: com_google_api_gax_java
+
+# Versions of oneself
+version.gax=1.40.1-SNAPSHOT
+version.gax_grpc=1.40.1-SNAPSHOT
+version.gax_bom=1.40.1-SNAPSHOT
+version.gax_httpjson=0.57.1-SNAPSHOT
+
+# Versions for dependencies which actual artifacts differ between Bazel and Gradle
```

Comment:
Can you comment on why this is needed? Are the version numbers here the ones that gradle, but not bazel, uses?

Reply:
Added